neuralprophet/forecaster.py

```diff
@@ -3021,15 +3006,17 @@ def _reshape_raw_predictions_to_forecst_df(self, df, predicted, components):
                         df_forecast = pd.concat([df_forecast, yhat_df], axis=1, ignore_index=False)
         return df_forecast
 
-    def conformalize(self, df_cal, alpha, method="naive", plotting_backend="default"):
-        """Apply a given conformal prediction technique to get the uncertainty prediction intervals (or q-hats).
+    def conformal_predict(self, df, alpha, calibration_df=None, method="naive", plotting_backend="default"):
```

Comment:
@karl-richter I believe you wanted to add a few more parameters to the regular `predict` function - not sure how we deal with multiple variants of the prediction function, we could replicate the additional parameters and pass them through, thoughts?

Reply:
Pass in **kwargs?

Reply:
Implemented.